adal/src/main/java/com/microsoft/aad/adal/AuthenticationRequestState.java

```diff
@@ -31,12 +31,15 @@
     private boolean mCancelled = false;
 
     private AuthenticationRequest mRequest = null;
-    
+
+    private final APIEvent mAPIEvent;
+
     public AuthenticationRequestState(int requestCallbackId, AuthenticationRequest request,
-            AuthenticationCallback<AuthenticationResult> delegate) {
+            AuthenticationCallback<AuthenticationResult> delegate, final APIEvent apiEvent) {
```

Comment:
lets discuss this tomorrow.

---
In reply to: [76169647](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/703#discussion_r76169647) [](ancestors = 76169647,76166000,76135343)